conda/meta.yaml

```diff
@@ -7,24 +7,27 @@ source:
     git_rev: v1.6.0
 
 build:
-    script: python setup.py install
+    number: 1
+    noarch: python
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
-  build:
-    - python {{ python }}
+  host:
+    - python
     - setuptools
+    - pip
 
   run:
-    - python {{ python }}
-    - cdp 1.4.1
-    - vcs 8.0
-    - vtk-cdat 8.0.1.8.0
-    - vcsaddons 8.0
-    - dv3d 8.0
-    - cdms2 3.0.1
-    - cdutil 8.0
-    - genutil 8.0
-    - cdtime 3.0
+    - python
+    - cdp >=1.4.1
+    - vcs >=8.0
+    - vtk-cdat 8.2.0rc2.8.1
+    - vcsaddons >=8.0
+    - dv3d >=8.0
+    - cdms2 >=3.0.1
+    - cdutil >=8.0
+    - genutil >=8.0
+    - cdtime >=3.0
```

Comment:
Because vtk-cdat is a release candidate, I have to pin it explicitly. All other versions, I changed to `>=` to allow the latest `cdat` to work.